indev/libinput_drv.h

```diff
@@ -36,11 +36,35 @@ extern "C" {
 /**********************
  *      TYPEDEFS
  **********************/
+enum libinput_capability {
```

Comment:
Usually we use `typedef enum`s. Can we use it here too?

Reply:
Ah, yes absolutely. That makes all the call sites a little more brief. :+1: 

Made the change.

Reply:
Thank you! :slightly_smiling_face: